src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,22 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+
+import Movie from '../MovieCard/MovieCard';
```

Comment:
```suggestion
import Movie from '../MovieCard';
```
Тут вроде бы можно так сократить, для большей красоты)